libs/utils/utils.js

```diff
@@ -592,14 +592,19 @@ export async function loadDeferred(area, blocks, config) {
   });
 }
 
-function loadPrivacy() {
+export function loadPrivacy() {
   window.fedsConfig = {
     privacy: {
       otDomainId: '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
-      footerLinkSelector: '[href="https://www.adobe.com/#openPrivacy"]',
     },
   };
   loadScript('https://www.adobe.com/etc.clientlibs/globalnav/clientlibs/base/privacy-standalone.js');
+
+  const privacyTrigger = document.querySelector('footer a[href$="#openPrivacy"]');
```

Comment:
nit: I'd make the selector look for a hashtag anywhere in the link 
`a[href*="#openPrivacy"]`
Just in case a descriptive attribute is added such as #_dns (do not style) or who knows what else might be used in the future. This makes everything a little more robust.